static/app/components/organizations/globalSelectionHeader/getParams.tsx

```diff
@@ -195,7 +202,7 @@ export function getParams(
       end: coercedPeriod ? null : dateTimeEnd,
       // coerce utc into a string (it can be both: a string representation from router,
       // or a boolean from time range picker)
-      utc: getUtcValue(utc),
+      utc: getUtcValue(pageUtc || utc),
```

Comment:
`pageUtc` should take precedence if it is explicitly `false`. 

```suggestion
      utc: getUtcValue(pageUtc ?? utc),
```